openprescribing/dmd/management/commands/import_dmd.py

```diff
@@ -295,6 +295,30 @@ def process_datafiles(source_directory):
                         insert_row(cursor, info, row_data)
 
 
+def update_product_references():
+    """Update FK references to DMDProducts.
+
+    We need to do this because sometimes a VMP's ID changes between releases of
+    the data.  When this happens, the VMP's previous ID is available in the
+    vpidprev column.
+
+    It is only VMPs that might change their ID, so we do not need to consider
```

Comment:
Do we know this because its only them that have a `idprev` field?

Reply:
Yes.